amundsen_application/static/js/components/common/SearchBar/styles.scss

```diff
@@ -45,16 +45,11 @@
       }
 
       &:focus {
-         // TODO: gray-60?
-        border-color: $gray-dark;
+        border-color: $stroke-focus;
       }
-    }
 
-    .search-bar-input:focus + .search-button,
-    .search-bar-input:focus + .search-button-small {
-      img.icon {
-        // TODO: gray-60?
-        background-color: $gray-dark;
+      &:focus + .search-button .icon-search:not(:hover) {
```

Comment:
Will this get applied to `.search-button-small`, as in when the `SearchBar` is in the header? Looks good otherwise.

Reply:
Works on both sizes. We actually don't use `.search-button-small`, but we have `.search-button small` instead.

Reply:
ah, good.